### PR TITLE
Add switch for dev/prod stop list generation

### LINKED ISSUE #166: View Issues throws SQLException
Receiving an exception after flipping through the application and returning to view_issues page.

## Detailed Description
SQLite3::SQLException: no such function: least: SELECT  1 AS one FROM "stops" WHERE (stops.lattitude IS NOT NULL AND stops.longitude IS NOT NULL) AND (stops.lattitude>{MY_LATITUDE} AND stops.lattitude<{MY_LATITUDE} AND stops.longitude>-{MY_LONGITUDE}AND stops.longitude<{MY_LONGITUDE}) AND ((
          (CASE WHEN stops.lattitude IS NULL OR stops.longitude IS NULL THEN NULL ELSE
          (ACOS(least(1,COS(0.7313864503829949)*COS(-1.5294003562686826)*COS(RADIANS(stops.lattitude))*COS(RADIANS(stops.longitude))+
          COS(0.7313864503829949)*SIN(-1.5294003562686826)*COS(RADIANS(stops.lattitude))*SIN(RADIANS(stops.longitude))+
          SIN(0.7313864503829949)*SIN(RADIANS(stops.lattitude))))*3963.1899999999996)
          END)
          <= 0.2)) LIMIT ?

## Steps to Reproduce

On sign in, the calculation works fine and I receive nearby stops. I then click in all sorts of different paths back and forth to the view_issues page and after 3-10 visits back I receive this error. Once the error starts it does not stop until I restart the server.

## Your Environment
* Environment name and version (e.g. Chrome 39, node.js 5.4): Chrome and Firefox (Once issue begins on one it will occur on the other, seems to be purely backend issue)
* Operating System and version (desktop or mobile): Windows 10

### PR DESCRIPTION
**Pull request by ajblaida**
Closes #166 - View Issues throws SQLException

### Description of Changes:
Add check on environment flag. If it is prod, we do the existing geolocation. If it is not, we simply grab the top 50 stops so we don't lose functionality.

### How This Was Tested:
Manual testing, rails test.

### Related Issue(s) or Specifications:
Closes #166 

### Checklist:
- [x] Code follows code style of this project
- [ ] Tests added to cover changes
- [x] All new and existing tests passing

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -2,9 +2,13 @@
 
 class PagesController < ApplicationController
   def view_issues
-    # If our location.js has pulled a location, use it to find Stops
-    if (params[:lat]) && (params[:long])
-      @nearby_stops = Stop.includes(:lines).within(0.2, origin: [params[:lat], params[:long]])
+	# If rails is not prod, just take the top 50 stops.
+	# There is a known issue with SQLite and the geolocation gem
+	if (Rails.env != "production")
```
Comment (vkoves) [inline app/controllers/pages_controller.rb]: There's already a nice function you can use for this - `Rails.env.production?`. See [env in the Rails docs](https://apidock.com/rails/Rails/env/class).

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -2,9 +2,13 @@
 
 class PagesController < ApplicationController
   def view_issues
-    # If our location.js has pulled a location, use it to find Stops
-    if (params[:lat]) && (params[:long])
-      @nearby_stops = Stop.includes(:lines).within(0.2, origin: [params[:lat], params[:long]])
+	# If rails is not prod, just take the top 50 stops.
```
Comment (vkoves) [inline app/controllers/pages_controller.rb]: It looks like you are using tabs for indents while rails (and the rest of this file) uses two space indents. Can you fix that? I'm surprised Rubocop didn't fail on that, so I might make some tweaks so CI will tell you that automatically

**Comment by vkoves** [comment]
@tkimia - I don't think this makes sense in a model (since geolocation is specific to this page only) though _maybe_ it could go in an initializer. I don't know how that would be architected though and this solution is light enough that I think the controller is a fine place for it to be :smiley:

**Comment by vkoves** [comment]
@ajblaida - one quick tip on using the PR template: To check a checkbox, write `[x]` in the markdown - that'll make it look like this:

- [X] Code follows code style of this project
- [ ] Tests added to cover changes
- [X] All new and existing tests passing

**Comment by ajblaida** [comment]
@vkoves thanks for the tip, updated!

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (ajblaida) [reply app/controllers/pages_controller.rb]: Looks better, thanks for the tip

File: app/controllers/pages_controller.rb (hunk 2 above)
Comment (ajblaida) [reply app/controllers/pages_controller.rb]: Fixed this and set in my workspace settings in the meantime

**Pull request merged**